src/pages/Products/Products.jsx

```diff
@@ -1,9 +1,293 @@
-import React from 'react'
+import React from "react";
+import ProductCard from "../../components/Card/ProductCard/ProductCard";
+import "./products.css";
+import {useEffect} from "react";
+import axios from "axios";
+import { useProducts } from "../../contexts/Providers/ProductProvider/ProductProvider";
+
 
 function Products() {
+  const {state, dispatch, filteredProducts} = useProducts();
+
+  const {
+    products,
+    category,
+    sortBy,
+    rating,
+    EMI,
+    filterPrice,
+  } = state;
+
+  useEffect(() => {
+    document.title = "Products | CarSmart";
+  }, []);
```

Comment:
You can *create a custom hook* for this as you might want to change the *document.title* for every page. ( kindof: **useDocumentTitle** )

Reply:
Yes, I'd be in need of that, I've added the custom hook. Please review